Address reported pylint issues

Either fix the reported issue or explicitly add an ignore to silence the warning if it's "written as intended".

**Issues:**
```
hawker@mbp:~/src/github.com/ahawker/ulid|master⚡
⇒  make lint
************* Module ulid
W: 11, 0: Wildcard import api (wildcard-import)
W: 13, 0: Wildcard import ulid (wildcard-import)
************* Module ulid.api
C: 21, 0: Invalid constant name "TimestampPrimitive" (invalid-name)
C: 27, 0: Invalid constant name "RandomnessPrimitive" (invalid-name)
************* Module ulid.hints
C: 12, 0: Invalid constant name "Buffer" (invalid-name)
************* Module ulid.ulid
C:191, 8: Invalid variable name "ms" (invalid-name)
make: *** [lint] Error 20
```

--- a/ulid/__init__.py
+++ b/ulid/__init__.py
@@ -8,9 +8,9 @@
     :license: Apache 2.0, see LICENSE for more details.
 """
 from . import api
-from .api import *
+from .api import *   # pylint: disable=wildcard-import
 from . import ulid
-from .ulid import *
+from .ulid import *  # pylint: disable=wildcard-import
 
 
 __all__ = api.__all__ + ulid.__all__

--- a/ulid/api.py
+++ b/ulid/api.py
@@ -18,13 +18,13 @@ __all__ = ['new', 'from_bytes', 'from_int', 'from_str', 'from_uuid', 'from_times
 
 #: Type hint that defines multiple primitive types that can represent
 #: a Unix timestamp in seconds.
-TimestampPrimitive = typing.Union[int, float, str, bytes, bytearray, memoryview,
+TimestampPrimitive = typing.Union[int, float, str, bytes, bytearray, memoryview,  # pylint: disable=invalid-name
                                   datetime.datetime, ulid.Timestamp, ulid.ULID]
 
 
 #: Type hint that defines multiple primitive types that can represent
 #: randomness.
-RandomnessPrimitive = typing.Union[int, float, str, bytes, bytearray, memoryview,
+RandomnessPrimitive = typing.Union[int, float, str, bytes, bytearray, memoryview,  # pylint: disable=invalid-name
                                    ulid.Randomness, ulid.ULID]
 
 

--- a/ulid/hints.py
+++ b/ulid/hints.py
@@ -9,4 +9,4 @@ import typing
 
 #: Type hint that defines multiple types that implement the buffer protocol
 #: that can encoded into a Base32 string.
-Buffer = typing.Union[bytes, bytearray, memoryview]
+Buffer = typing.Union[bytes, bytearray, memoryview]  # pylint: disable=invalid-name

--- a/ulid/ulid.py
+++ b/ulid/ulid.py
@@ -188,8 +188,8 @@ class Timestamp(MemoryView):
         :return: Timestamp in datetime form.
         :rtype: :class:`~datetime.datetime`
         """
-        ms = self.int
-        return datetime.datetime.utcfromtimestamp(ms // 1000.0).replace(microsecond=ms % 1000 * 1000)
+        mills = self.int
+        return datetime.datetime.utcfromtimestamp(mills // 1000.0).replace(microsecond=mills % 1000 * 1000)
 
 
 class Randomness(MemoryView):

